support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -325,6 +325,7 @@ function WeeklyCheckoutForm(props: PropTypes) {
 						onChange={(billingPeriod) => props.setBillingPeriod(billingPeriod)}
 						billingPeriods={weeklyBillingPeriods(
 							props.participations.sixForSixSuppression !== 'variant',
+							props.deliveryCountry === 'US' || props.deliveryCountry === 'CA',
```

Comment:
We have this `props.deliveryCountry === 'US' || props.deliveryCountry === 'CA'` (or similar) logic in 2 places: `weeklyCheckoutForm.tsx` and `weeklyProductPrices.tsx`. A potential issue with this is if we want to add quarterly to another country we'd need to know to update 2 places in the code. 

As an alternative we could move this logic into the `weeklyBillingPeriods` function in `billingPeriods.ts`, we'd need to change the signature of the function to take an `IsoCountry` as it's 2nd argument rather than a `boolean`. This means there's only one place in the code we'd need to update if we wanted to introduce/remove quarterly to/from other countries in the future.